lib/database/ticket.php

```diff
@@ -471,12 +494,45 @@ function countOpenTickets(
         return 0;
     }
 
+    // Developer Active condition
+    $devJoin = "";
+    $devActiveCond = getDevActiveCondition($ticketFilters);
+    if ($devActiveCond === null) {
+        return 0;
+    } elseif ($devActiveCond != "") {
+        $devJoin = "LEFT JOIN UserAccounts AS ua3 ON ua3.User = ach.Author";
+    }
+
+    // Karama condition
+    $resolverJoin = "";
+    $karmaCond = getKarmaCondition($ticketFilters);
+    if ($karmaCond != "") {
+        $resolverJoin = "LEFT JOIN UserAccounts AS ua2 ON ua2.ID = tick.ResolvedByUserID";
+    }
+
     // Author condition
     $authorCond = "";
     if ($assignedToUser != null) {
         $authorCond = " AND ach.Author LIKE '$assignedToUser'";
     }
 
+    // Reporter condition
+    $reporterCond = "";
+    $reporterJoin = "";
+    if ($reportedByUser != null) {
+        $reporterJoin = "LEFT JOIN UserAccounts AS ua ON ua.ID = tick.ReportedByUserID";
+        $reporterCond = " AND ua.User LIKE '$reportedByUser'";
+    }
+
+    // Resolver condition
+    $resolverCond = "";
+    if ($resolvedByUser != null) {
+        $resolverCond = " AND ua2.User LIKE '$resolvedByUser'";
```

Comment:
Yep you're right, I added it to $resolverJoin instead and also swapped that check from the karmaCond check into the $resolverJoin it sets since it used that same join statement.